script/DeployCompoundGovernor.s.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// slither-disable-start reentrancy-benign
+
+pragma solidity 0.8.26;
+
+import {Script} from "forge-std/Script.sol";
+import {TransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import {IVotes} from "@openzeppelin/contracts/governance/utils/IVotes.sol";
+import {ICompoundTimelock} from "@openzeppelin/contracts/vendor/compound/ICompoundTimelock.sol";
+import {CompoundGovernor} from "contracts/CompoundGovernor.sol";
+import {CompoundGovernorConstants} from "script/CompoundGovernorConstants.sol";
+
+// Deploy script for the underlying implementation that will be used by both Governor proxies
+contract DeployCompoundGovernor is Script, CompoundGovernorConstants {
+    uint256 deployerPrivateKey;
+
+    function setUp() public virtual {
+        // private key of the deployer (use the anvil default account #0 key, if no environment variable is set)
+        deployerPrivateKey = vm.envOr(
```

Comment:
is it possible to avoid setting a PK in the env? Would prefer to let foundry manage the wallet instead. We run across this often, and i remember there's a limitation with calling scripts in tests. maybe this is something we can create an issue for.

Reply:
Added an issue: https://github.com/ScopeLift/compound-governance-upgrade/issues/18